Apps/Sandcastle/gallery/development/Polygon Texture Coordinates.html

```diff
@@ -0,0 +1,443 @@
+<!DOCTYPE html>
```

Comment:
Could you add some of the polygons from this sandcastle to "Geometry and Appearances" sandcastle? That way there a non-development Sandcastle using this feature that we can look at when doing releases and there's example usage in the entity API.


"Development/Geometry and Appearances" could be updated too just so they stay in sync with each other

Reply:
Add an extruded textured polygon and a textured polygon with holes to both Sandcastles. Also, updated the thumbnails for both the Sandcastles.